client/src/ui/track/JobCard.tsx

```diff
@@ -0,0 +1,186 @@
+
+  // Handle status change and update the card's state
+  const handleStatusChange = (newStatus: SetStateAction<ApplicationStage>) => {
+    const resolvedStatus =
+      typeof newStatus === "function" ? newStatus(Status) : newStatus;
+    if (resolvedStatus !== Status) {
+      moveApplication(Status, resolvedStatus, applicationId);
+    }
+  };
+
+  return (
+    <Card
+      sx={{
+        borderRadius: "8px",
+        border: "2px solid #E0E4F2",
+        backgroundColor: "#F6F8FF",
+        display: "grid",
+        gridTemplateColumns: "50px auto 30px", // Fixed width for each column
+        alignItems: "center",
+        padding: "10px", // Consistent padding around the card
+        marginX: "auto",
+        boxShadow: "none",
+        marginBottom: "10px", // Space between cards
+        overflow: "hidden", // Ensures content doesn’t overflow
+      }}
+    >
+      {/* Column 1: Company Brand */}
+      <Box
+        sx={{
+          display: "flex",
+          alignItems: "center",
+          justifyContent: "center",
+          padding: "4px",
+        }}
+      >
+        <Image
+          src="/landingpage/logo.svg"
+          height={40}
+          width={40}
+          alt="Squeakpeek Logo"
+          style={{
+            objectFit: "cover",
+            borderRadius: "8px",
+          }}
+        />
+      </Box>
+
+      {/* Column 2: Main Content */}
+      <Box
+        sx={{
+          display: "flex",
+          flexDirection: "column",
+          justifyContent: "center",
+          alignItems: "flex-start",
+          gap: "4px",
+          ml: 1,
+        }}
+      >
+        {/* Row 1: Company Name and Status */}
+        <Box
+          sx={{
+            display: "flex",
+            alignItems: "center",
+            gap: "4px",
+            width: "100%",
+          }}
+        >
+          <Typography variant="subtitle2" sx={{ fontSize: "12px" }}>
+            {Company}
+          </Typography>
+          <UpdateStatus
+            required
+            name="status"
+            options={[
+              "Applied",
+              "Rejected",
+              "Online Assesstment",
+              "Interviewing",
+              "Offer",
+            ]}
+            applicationStatus={Status}
+            setApplicationStage={handleStatusChange}
+            customSx={{
+              height: "20px",
+              fontSize: "10px",
+              width: "70px", // Adjust width for fit
+            }}
+          />
+        </Box>
+
+        {/* Row 2: Role Title */}
+        <Typography variant="subtitle2" sx={{ fontSize: "12px" }}>
+          {Role}
+        </Typography>
+
+        {/* Row 3: Icon Buttons */}
+        <Box sx={{ display: "flex", gap: "6px" }}>
+          <IconButton
+            sx={{
+              padding: "6px", // Adjusted padding for larger button size
+              borderRadius: "50%",
+            }}
+          >
+            <FontAwesomeIcon
+              icon={faFacebookMessenger}
```

Comment:
Can we change this to the conversation icon instead to make it consistent with the UI on the navbar

![image](https://github.com/user-attachments/assets/74a37ea3-5f6f-4131-a65d-4974b71e2890)